src/en-GB/news/blog/2018/ansible-module-to-manage-cephx-keys/index.md

```diff
@@ -0,0 +1,31 @@
n></span><br><span class="line"><span class="attr">        name:</span> <span class="string">"<span class="template-variable">{{ item.name }}</span>"</span></span><br><span class="line"><span class="attr">        state:</span> update</span><br><span class="line"><span class="attr">        caps:</span> <span class="string">"<span class="template-variable">{{ item.caps }}</span>"</span></span><br><span class="line"><span class="attr">        cluster:</span> <span class="string">"<span class="template-variable">{{ cluster }}</span>"</span></span><br><span class="line"><span class="attr">      with_items:</span> <span class="string">"<span class="template-variable">{{ keys_to_create }}</span>"</span></span><br><span class="line"></span><br><span class="line"><span class="attr">    - name:</span> delete ceph key(s)</span><br><span class="line"><span class="attr">      ceph_key:</span></span><br><span class="line"><span class="attr">        name:</span> <span class="string">"<span class="template-variable">{{ item }}</span>"</span></span><br><span class="line"><span class="attr">        state:</span> absent</span><br><span class="line"><span class="attr">        cluster:</span> <span class="string">"<span class="template-variable">{{ cluster }}</span>"</span></span><br><span class="line"><span class="attr">      with_items:</span> <span class="string">"<span class="template-variable">{{ keys_to_delete }}</span>"</span></span><br><span class="line"></span><br><span class="line"><span class="attr">    - name:</span> info ceph key(s)</span><br><span class="line"><span class="attr">      ceph_key:</span></span><br><span class="line"><span class="attr">        name:</span> <span class="string">"<span class="template-variable">{{ item }}</span>"</span></span><br><span class="line"><span class="attr">        state:</span> info</span><br><span class="line"><span class="attr">        cluster:</span> <span class="string">"<span class="template-variable">{{ cluster }}</span>"</span></span><br><span class="line"><span class="attr">      register:</span> key_info</span><br><span class="line"><span class="attr">      ignore_errors:</span> <span class="literal">true</span></span><br><span class="line"><span class="attr">      with_items:</span> <span class="string">"<span class="template-variable">{{ keys_to_info }}</span>"</span></span><br><span class="line"></span><br><span class="line"><span class="attr">    - name:</span> list ceph key(s)</span><br><span class="line"><span class="attr">      ceph_key:</span></span><br><span class="line"><span class="attr">        state:</span> list</span><br><span class="line"><span class="attr">        cluster:</span> <span class="string">"<span class="template-variable">{{ cluster }}</span>"</span></span><br><span class="line"><span class="attr">      register:</span> list_keys</span><br><span class="line"><span class="attr">      ignore_errors:</span> <span class="literal">true</span></span><br></pre></td></tr></tbody></table>
```

Comment:
Double curly brackets are causing issues as this syntax is used with nunjucks. In this instance it's causing the content between the brackets not to appear.

Can they be escaped?

E.g.
`\{\{ content here \}\}`

This file contains multiple instances of double curly brackets.

Reply:
Fixed